tests/unit/test.program.js

```diff
@@ -323,4 +324,13 @@ describe('program.defaultVersionGetter', () => {
       });
   });
 
+  it('returns the development package version', () => {
+    process.env.NODE_ENV = 'development';
```

Comment:
If you remove the modifications to this global object then you won't need to set this env var here